erts/emulator/beam/beam_load.c

```diff
@@ -6332,7 +6355,7 @@ erts_make_stub_module(Process* p, Eterm Mod, Eterm Beam, Eterm Info)
 	goto error;
     }
     define_file(stp, "atom table", ATOM_CHUNK);
-    if (!load_atom_table(stp)) {
+    if (!load_atom_table(stp, ERTS_ATOM_ENC_LATIN1)) {
```

Comment:
Thank you. I have run the code suite and fixed this and the remaining failing tests.